swift/ql/test/extractor-tests/generated/expr/TypeValueExpr/TypeValueExpr.expected

```diff
@@ -0,0 +1,2 @@
+| type_value_exprs.swift:4:13:4:13 | TypeValueExpr | hasType: | yes | getTypeRepr: | type_value_exprs.swift:4:13:4:13 | (no string representation) |
+| type_value_exprs.swift:5:13:5:13 | TypeValueExpr | hasType: | yes | getTypeRepr: | type_value_exprs.swift:5:13:5:13 | (no string representation) |
```

Comment:
I'm not sure why I'm seeing `(no string representation)` here

Reply:
yeah, that is odd, I thought there was a default implementation in place (using the primary class). We can look at it in a follow up I think